docs/docsite/rst/dev_guide/testing/sanity/import.rst

```diff
@@ -1,5 +1,69 @@
 import
 ======
 
-All Python imports in ``lib/ansible/modules/`` and ``lib/ansible/module_utils/`` which are not from the Python standard library
-must be imported in a try/except ImportError block.
+All Python imports in the directories specified that are not from a limited set of possibilities listed below
+must be imported in a try/except ImportError block as follows:
```

Comment:
Without the specific directories, I find this a little confusing. It's also passive voice. What about this?
```suggestion
Ansible allows unchecked imports of some libraries from specific directories, listed at the bottom of this section. Import all other Python libraries in a try/except ImportError block as follows:
```
Adding a short sentence about why this is useful/important/required would help make the rule more memorable.

Reply:
@acozine I added an explanation as well: 11d2f5f2a5ecbd6e3f117c7b7580597bc3751123